Vagrantfile

```diff
@@ -0,0 +1,22 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+Vagrant.configure(2) do |config|
+  config.vm.box = "ubuntu/trusty64"
+  config.vm.network "forwarded_port", guest: 8123, host: 8123
+  config.vm.synced_folder ".", "/vagrant"
+  config.vm.provision "shell", inline: <<-SHELL
```

Comment:
vagrant wont replace virtualenv. 
Docker, vagrant, and virtualenv can happily coexist since different developers will be more comfortable with different tools. 

For me personally, I've never used virtualenv, and I don't want to install python3 on my mac if I don't have to. I'm sure there are others who might not have ever used docker or virtualenv, but are comfortable with vagrant.